### PR TITLE
docs(emitter): adjust documentation for Emitter component

### PR DESCRIPTION
**Pull request by willamesoares**
Add more examples and sections to the documentation page for the `Emitter` component.
Use the `notification` element to style relevant messages.
Since this page does not have too many sections, it is not necessary to create a summary for an easier navigation.

### CODE REVIEW

File: docs/js/emitter.md [hunk 1]
```diff
@@ -8,27 +8,43 @@ path: emitter
...
+To register an event you can use the `emitter.on()` function, passing the event name and a handler as parameters.
 ```js
 import emitter from 'src/utils/emitter';
 
+emitter.on('eventName', () => {
+  // Here you can handle the event.
+});
+```
+### Passing arguments
+It is possible to pass arguments to a listener while emitting an event with the `emitter.emit()` function.
```
Comment (felipesoares6) [inline docs/js/emitter.md]: It's ?

Comment (willamesoares) [inline docs/js/emitter.md]: I personally don't like using the contract form in cases like this. It sounds clearer to me this way, but let's see what others have to say :)

File: docs/js/emitter.md [hunk 2]
```diff
@@ -8,27 +8,43 @@ path: emitter
...
+```js
+import emitter from 'src/utils/emitter';
 
-emitter.on('validation:error', (field, errors) => {
-  // Here you can handle an errror event, emitted by the validation component.
+emitter.on('event', (data) => {
+  // Here you can work with the data passed.
```
Comment (vitortalaia) [inline docs/js/emitter.md]: Maybe `received data` would fit better here, what do you think?

Comment (willamesoares) [inline docs/js/emitter.md]: Yes! Much better, thanks :)

**Pull request closed without merging**